apps/spruce/src/pages/waterfall/index.tsx

```diff
@@ -36,17 +37,28 @@ const Waterfall: React.FC = () => {
   const pageWrapperRef = useRef<HTMLDivElement>(null);
   const { atTop } = useIsScrollAtTop(pageWrapperRef, 200);
 
+  const guideCueRef = useRef<WalkthroughGuideCueRef>(null);
+
   return (
     <>
       <Global styles={navbarStyles} />
       <PageContainer ref={pageWrapperRef} data-cy="waterfall-page">
-        {isBeta() && (
-          <Banner>
-            <strong>Thanks for using the Waterfall Alpha!</strong> Feedback?
-            Open a ticket within the project epic{" "}
-            {jiraLinkify("DEVPROD-3976", jiraHost ?? "")}.
-          </Banner>
-        )}
+        <Banner>
+          <BannerContent>
+            <div>
+              <strong>Thanks for using the Waterfall Alpha!</strong> Feedback?
+              Open a ticket within the project epic{" "}
+              {jiraLinkify("DEVPROD-3976", jiraHost ?? "")}.
+            </div>
+            <Button
+              data-cy="restart-walkthrough-button"
+              onClick={() => guideCueRef.current?.restart()}
+              size={ButtonSize.XSmall}
+            >
+              Restart walkthrough
+            </Button>
```

Comment:
Hrmmmm I would love to show this button and launch the walkthrough only once the beta period starts. Is that doable now or is it dependent on some of the other tix?

Reply:
i think it might be simpler if i merge it on friday or monday...? 🤔 im on deploy so i can wait for whatever dependencies and deploy next week if that makes sense

Reply:
Make sense to me! We can throw a `do-not-merge` label on here

Reply:
Make sense to me! We can throw a `do-not-merge` label on here